Library/Formula/mariadb.rb

```diff
@@ -146,6 +139,8 @@ def install
         s.gsub!("$(dirname $0)/wsrep_sst_common", "#{libexec}/wsrep_sst_common")
       end
     end
+    # remove non-executable from bin
+    rm bin+"maria_add_gis_sp.sql"
```

Comment:
I checked for usages of this file and found none. There is still a copy of this file located in ```share/mysql/```.
```mysql_install_db``` only uses ```maria_add_gis_sp_bootstrap.sql``` which has the exact same functionality.